lib/bundler/ruby_version.rb

```diff
@@ -38,6 +38,24 @@ def to_s(versions = self.versions)
       output
     end
 
+    # @private
+    PATTERN = %r{
+      ruby\s
```

Comment:
pretty sure the "ruby " part of this is supposed to be optional? I see a lot of .ruby-version files that are only, say, "2.2.3".